tardis/widgets/sdec_plot.py

```diff
@@ -876,10 +876,14 @@ def generate_plot_mpl(
         self.ax.set_xlabel(r"Wavelength $[\AA]$", fontsize=12)
         if distance:  # Set y-axis label for flux
             self.ax.set_ylabel(
-                r"$F_{\lambda}$ [erg/s/cm^{2}/$\AA$]", fontsize=12
+                r"Flux [erg $\mathrm{s^{-1}}$ $\mathrm{cm^{-2}}$ $\mathrm{\AA^{-1}}$]",
+                fontsize=12,
```

Comment:
There seems to be some confusion about what to change on the axis labels - I didn't help things with a vague comment, but I am not proposing any change to the actual label - just that the Angstrom unit is no longer italicised. The comment posted was just grabbed from one of my programs illustrating how to use mathrm. Can this be reverted back to F_lambda?

Reply:
Yes, No Issues but the F_lambda & L_lambda will be italicised ... I will update the screenshots with the same :)

Reply:
Flux
![image](https://user-images.githubusercontent.com/66117751/107249676-95a23000-6a59-11eb-9372-92a28a47a594.png)